pynicotine/nowplaying.py

```diff
@@ -198,9 +201,16 @@ def mpris(self, player):
                 log.add_warning(_("Auto-detected MPRIS player: %s."), player)
```

Comment:
@mathiascode, is there not another method on the logger for informational messages as an alternative to relying on a warning?

Reply:
Good catch, this should be a normal log message (add) instead of a warning (add_warning).